src/main/kotlin/hu/kirdev/schpincer/service/HibernateSearchService.kt

```diff
@@ -21,9 +21,9 @@ open class HibernateSearchService(entityManagerFactory: EntityManagerFactory) {
 
 	private val log = LoggerFactory.getLogger(javaClass)
 
-    val entityManager: EntityManager
+    val entityManager: EntityManager = entityManagerFactory.createEntityManager()
 
-    @Autowired
+	@Autowired
```

Comment:
Unnecessary indentation.

Reply:
This somehow changed to a tab instead of 4 spaces, this is why it looks longer from here.